pulp_rpm/app/serializers/repository.py

```diff
@@ -270,6 +259,15 @@ class RpmRemoteSerializer(RpmBaseRemoteSerializer):
         allow_null=True,
     )
 
+    def validate_url(self, value):
```

Comment:
~~You could pull this logic out into a `SchemeValidationMixin` class inherited by both `RpmRemoteSerializer` and `UlnRemoteSerializer`, and move the definition of `ALLOWED_SCHEMES` to the remote class rather than the function.~~

~~That would deduplicate the logic~~

on second thought, while you could do that, it probably isn't worth it just to deduplicate about 4 lines of code

Reply:
I wouldn't mind, but we should be fine